cypress/integration/home.spec.ts

```diff
@@ -1,9 +1,21 @@
 /// <reference path="../support/index.d.ts" />
 
-// import fake from 'faker'
+describe('Test desktop element visibility', () => {
+  beforeEach(() => {
+    cy.clearLocalStorage()
+    cy.visit('/')
+  })
 
-describe('Desktop Login Test', () => {
-  // const email = fake.internet.email()
+  it('smiley should wink on mouseover', () => {
+    cy.get('.face')
+      .contains(':)')
+    cy.get('.face')
+      .trigger('mouseover')
+      .contains(';)')
+  })
+})
+
+// const email = fake.internet.email()
```

Comment:
We can delete these until we want to integrate them later

Reply:
Deleted
6b5376a